sunpy/net/jsoc/jsoc.py

```diff
@@ -213,13 +221,26 @@ def query(self, *query, **kwargs):
             iargs = kwargs.copy()
             iargs.update(block)
             blocks.append(iargs)
-
             return_results.append(self._lookup_records(iargs))
 
         return_results.query_args = blocks
-
         return return_results
 
+    def get_metadata(self, *query, **kwargs):
+
+        query = and_(*query)
+        blocks = []
+        dicts = {}
+        for block in walker.create(query):
+            iargs = kwargs.copy()
+            iargs.update(block)
+            iargs.update({'meta': True})
+            blocks.append(iargs)
+            r = self._lookup_records(iargs)
```

Comment:
How time-dependant is the result? If it's likely the user will want to work on it like a time series (which I think from what Kolja was saying they will) a Dataframe would probably be the best.